examples/data/cleansed/metadata.yml

```diff
@@ -0,0 +1,61 @@
+scada:
+  time: time
+  WTUR_TurNam: WTUR_TurNam
+  WTUR_W: WTUR_W
+  WMET_HorWdSpd: WMET_HorWdSpd
+  WMET_HorWdDir: WMET_HorWdDir
+  WTUR_TurSt: WTUR_TurSt
+  WROT_BlPthAngVal: WROT_BlPthAngVal
+  WMET_EnvTmp: WMET_EnvTmp
+  WTUR_SupWh: WTUR_SupWh
```

Comment:
Should WTUR_SupWh be removed here, since it gets derived within `PlantData`?

Reply:
From my above comment, the idea is more or less to archive the data that was used so it can be used all over again. In this case, the column will be recomputed because of the decision to make energy internally calculated rather than externally provided.

Reply:
OK, that make sense then